index.js

```diff
@@ -18,27 +18,79 @@ console.log("Java Script has loaded successfully");
 //Present  check
 // Validation that hours are correct for the level
 
-// Capture user's input on form submission
 let form = document.querySelector("form");
 
 form.addEventListener("submit", function (event) {
   event.preventDefault();
-  console.log("Form submitted");
+
+  const maxHoursPerLevel = {
+    basic: 5,
+    advanced: 10,
+  };
+
+  // Store the user's email address as userEmail (string/text)
+  // Capture user's input on form submission
+
+  // Store the user's email address as userEmail (string/text)
+  let userEmail = document.querySelector("#email").value;
+
+  // Store the user's hours of study as userHours (number)
+  let userHours = parseInt(document.querySelector("#hoursPerWeek").value);
+
+  // Store the user's level as userLevel (string/text)
+  let userLevel = document.querySelector("#level").value;
+
+  // Validate the user's input
+
+  // Check if the user has provided an email address
+  if (userEmail === "") {
+    alert("Please enter your email address.");
+
+    return;
+  }
+
+  // Validate the user's input
+  // Check if the user has provided an email address
+  if (userLevel === "") {
+    alert("Please enter your level.");
+
+    return;
+  }
+
+  // Validate the user's input
+  // Check if the user has provided an email address
+  console.log(typeof userHours);
+  console.log(userHours);
+  if (isNaN(userHours) || userHours <= 0) {
+    alert("Please enter the number of hours.");
+
+    return;
+  }
+
+  if (!maxHoursPerLevel.hasOwnProperty(userLevel)) {
+    alert("In correct about of study for selected level");
+
+    return;
+  }
+
+  // Is userHours in range?
+  const maxAllowedHours = maxHoursPerLevel[userLevel];
+  if (userHours > maxAllowedHours) {
+    alert(`You can only study a maximum of ${maxAllowedHours} hours per week.`);
+
+    return;
+  }
+
+  console.log({ userEmail, userLevel, userHours });
```

Comment:
Good start. You are displaying alert messages successfully to the user if they submit invalid data.

It looks as though you are ready to start the [error handling](https://gedsargent.github.io/code-monster/guides/programming/harmony-music/error-handling/) tutorial on Code Monster 🚀